src/app/api/admin/feature-flags/route.ts

```
@@ -0,0 +1,176 @@
+        { status: 403 }
+      );
+    }
+
+    const body = await request.json();
+    const { action, flagKey, updates } = body;
+
+    const service = getFeatureFlagService();
+
+    switch (action) {
+      case 'toggle':
+        const toggleSuccess = service.toggleFlag(flagKey);
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)